src/pudl/etl.py

```diff
@@ -63,7 +63,7 @@ def _read_static_tables_eia() -> dict[str, pd.DataFrame]:
     return {
         table_name: code_dict["df"]
         for (table_name, code_dict) in CODE_METADATA.items()
-        if "_eia" in table_name  # right now th
+        if "_eia" in table_name  # only grab the eia tables
```

Comment:
I **think** what you're trying to do here can be accomplished without the implicit naming dependency by selecting all of the tables that have `etl_group=="static_eia"` (and something similar should be doable for the static FERC 1 tables)

I thought I had already implemented this. It was kind of the reason to create the static table ETL groups. But apparently not!  `etl_group` is a `Resource` attribute.

Also it could be that there are static tables which are not coding tables, so that would be more general.

Reply:
We also don't always load all of these tables. For instance we weren't previously loading the `entity_types_eia` table because it contains codes from both the EIA-860 and the EIA-861 which have not yet been reconciled (are they referring to the same kinds of entities? Is there overlap between the codes? Are they separate categorizations?)

I think I'd be inclined to keep it an enumeration here. Or to come up with a more robust way of obtaining the static tables that need to be loaded, and applying it across all the data sources.

Reply:
What if we used the etl_group, and simply removed this currently unused table? If it is not being loaded into the db it’s not truly in an etl group.

Reply:
Yeah I think that makes sense. You mean remove it from the `etl_group` not remove it entirely right?

Reply:
I see a few other tables being categorized as `ferc1_disabled`  I’m going to push a change that re-labels this one static table as `static_eia_disabled`. This enables a standardization of how we are grabbing static tables for both eia and ferc. It seems like a win.